frontend/src/containers/layout.js

```diff
@@ -52,47 +57,132 @@ class CustomLayout extends React.Component {
                 this.inboxMsg = this.props.inbox.items.items;
             }
         }
+
+        if (this.props.isAuthenticated){
+            this.props.getMyProfile();
```

Comment:
Why adding this line?

Reply:
This line will send too many requests, need more conditions before dispatch actions.

remove it if not need